CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.26.0 (2020-06-30)
+
+- Added additional domian to the auth domain whitelist. 
```

Comment:
The first instance of "domain" is spelled incorrectly. Also, now that you've created the PR, you can add a link to it at the end of the release note (see the previous release note for an example, and don't forget to credit yourself!)